linkedlist.c

```diff
@@ -0,0 +1,71 @@
+//erzeugt listenelement
+int* create(int* next, int data) {
+	int* s;
+	s = (int*)malloc( 2 * 4 );
+
+	// next == 0 entspricht dem letzten element, ansonsten pointer aufs naechste
+	*(s+0) = next;
+
+	// payload, daten kommen hier rein, zB 'a'
+	*(s+1) = data;
+
+	return s;
+}
+
+// entfernt element von liste
+// sucht element in liste rekursiv
+void remove(int* head, int* element) {
```

Comment:
Be careful with metrics like "easier to read". They are highly subjective and therefore hard to compare. The actual reason why recursion is more elegant here is the list itself. A linked list is a recursive data type. Nice solution!